installer/templates/phx_single/mix.exs

```diff
@@ -50,7 +50,8 @@ defmodule <%= @app_module %>.MixProject do
       {:phoenix_live_dashboard, github: "phoenixframework/phoenix_live_dashboard"},<% end %><%= if @assets do %>
       {:esbuild, "~> 0.5", runtime: Mix.env() == :dev},
       {:tailwind, "~> 0.1.8", runtime: Mix.env() == :dev},<% end %><%= if @mailer do %>
-      {:swoosh, "~> 1.3"},<% end %>
+      {:swoosh, "~> 1.3"},
+      {:finch, "~> 0.13.0"},<% end %>
```

Comment:
I think we can relax it a bit, especially if Swoosh requires a particular Finch version. :)

```suggestion
      {:finch, "~> 0.13"},<% end %>
```

Reply:
Addressed in [cacf1b5](https://github.com/phoenixframework/phoenix/pull/5052/commits/cacf1b5523d09327aa5469b393818b68facf25cd)